index.bs

```diff
@@ -8786,6 +8848,7 @@ These recommendations serve to prevent an adversary with physical access to an [
 
 ## Privacy considerations for [=clients=] ## {#sctn-privacy-considerations-client}
 
+
```

Comment:
Please amend the commit to remove the spurious whitespace changes here and on lines 325, 368, 8378 and 8477.